clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackProviderFactory.groovy

```diff
@@ -28,7 +28,7 @@ class OpenstackProviderFactory {
     OpenstackComputeV2Provider computeProvider = new OpenstackComputeV2Provider(identityProvider)
     OpenstackNetworkingProvider networkingProvider = new OpenstackNetworkingV2Provider(identityProvider)
     OpenstackOrchestrationProvider orchestrationProvider = new OpenstackOrchestrationV1Provider(identityProvider)
-    OpenstackImageProvider imageProvider = new OpenstackImageV1Provider(identityProvider)
+    OpenstackImageProvider imageProvider = new OpenstackImageV2Provider(identityProvider)
```

Comment:
The `OpenstackImageV1Provider` is still in Clouddriver, but not being used. Please either have this factory select either v1 or v2 based on something or just remove the v1 provider so we don't have unused classes.

Reply:
Done